db/seeds/cities.seeds.rb

```diff
@@ -0,0 +1,15 @@
+city_list = [
+  ['Lyon', '69000', '69', 'xxx', 'fr', 45.759723, 4.842223, 1],
+  ['Paris', '75000', '75', 'xxx', 'fr', 48.856578, 2.351828, 1],
+  ['Angers', '49000', '49', 'xxx', 'fr', 47.47361, -0.55416, 1],
+  ['Lille', '59000', '59', 'xxx', 'fr', 50.637222, 3.063333, 1],
+  ['Toulouse', '31000', '31', 'xxx', 'fr', 43.604482, 1.443962, 1],
+  ['Marseille', '13001', '13', 'xxx', 'fr', 43.296346, 5.369889, 1]
+]
+
+city_list.each do |name, postal_code, department, region, country_code, lat, lon, distance|
+  City.create(name: name, postal_code: postal_code, \
```

Comment:
Might be better to make it re-entrant via :
`City.where(postal_code: postal_code).first_or_initialize.update(name: name, ...)`